jsx/Panel.js

```diff
@@ -65,7 +76,11 @@ Panel.propTypes = {
   title: React.PropTypes.string
 };
 Panel.defaultProps = {
+  initCollapsed: false,
   id: 'default-panel',
-  height: '100%',
-  title: 'Selection Filter'
+  height: '100%'
 };
+
+window.Panel = Panel;
```

Comment:
If nothing is requiring it, I'd prefer to remove it and not pollute the global/window namespace.